Change get_started to recommened Gz version per platform for installations (#338)
gazebosim/docs: get_started.md

## Step 1: Install

> The recommended installation for non expert users is the use of binary

Reviewer: ```suggestion
The recommended installation for new users is the use of binary
```

packages available for the platform to use:

|Platform|Gz Versions|
|---|---|
| Ubuntu 22.04 Jammy | [Gz Garden](docs/garden/install_ubuntu) (recommended) and [Gz Fortress](docs/fortress/install_ubuntu)

> | Ubuntu 20.04 Focal | [Gz Garden](docs/garden/install_buuntu) (recommended), [Gz Fortress](docs/fortress/install_ubuntu) and [Gz Citadel](docs/citadel/install_ubuntu)

Reviewer 2: ```suggestion
| Ubuntu 20.04 Focal | [Gz Garden](docs/garden/install_ubuntu) (recommended), [Gz Fortress](docs/fortress/install_ubuntu) and [Gz Citadel](docs/citadel/install_ubuntu)
```

| Ubuntu 18.04 Bionic | [Gz Citadel](docs/citadel/install_ubuntu)
| Mac Monterey | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
| Mac BigSur | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)
| Mac Catalina | [Gz Garden](docs/garden/install_osx) (recommended), [Gz Fortress](docs/fortress/install_osx) and [Gz Citadel](docs/citadel/install_osx)

Windows support via conda-forge is not fully functional, there are known runtime problems

> [stoping Gz to work](https://github.com/gazebosim/gz-sim/issues/168).

Reviewer 2: ```suggestion
[stopping Gz to work](https://github.com/gazebosim/gz-sim/issues/168).
```

> [stopping Gz to work](https://github.com/gazebosim/gz-sim/issues/168).

Reviewer: ```suggestion
[Gz for Windows Issue](https://github.com/gazebosim/gz-sim/issues/168).
```

> If the desired platform is not listed above or if a particular feature in a

Reviewer: ```suggestion
If the desired platform is not listed above, or if a particular feature in a
```

> If the desired plaform is not listed above or if a particular feature in a
> given Gz release is needed, there is a installation package per release

Reviewer 2: ```suggestion
If the desired platform is not listed above or if a particular feature in a
given Gz release is needed, there is an installation package per release
```
